deploy/docker/docker-compose.yml

```diff
@@ -65,5 +66,63 @@ services:
       - ogree_api
     restart: on-failure:10
 
+
+
+  arango_db:
+    image: arangodb/arangodb:3.11.2
+    container_name: ${COMPOSE_PROJECT_NAME}_arango_db
+    environment:
+        - ARANGO_ROOT_PASSWORD=${ARANGO_PASS}
+    volumes:
+      - arangodb-persist:/var/lib/arangodb3
+    ports:
+      - ${ARANGO_PORT}:8529
+    restart: on-failure:10
+    
+
+  arango_api:
+    build:
+      context: ${CORE_DIR} 
+      dockerfile: ${ARANGO_API_BUILD_DIR}/Dockerfile
+    image: arango-api:${IMAGE_TAG}
+    restart: always
+    container_name: ${COMPOSE_PROJECT_NAME}_arango_api
+    environment:
+      - ENV=production
+      - ARANGO_URL=http://${COMPOSE_PROJECT_NAME}_arango_db:${ARANGO_PORT}
+      - ARANGO_DATABASE=${ARANGO_DB}
+      - ARANGO_USER=${ARANGO_USER}
+      - ARANGO_PASSWORD=${ARANGO_PASS}
+      - API_SECRET=${AUTH_SECRET}
+      - API_PORT=${ARANGO_API_PORT}
+    healthcheck:
+        test: ["CMD", "curl","-f","http://${COMPOSE_PROJECT_NAME}_arango_db:5829/_api/version"]
+        timeout: 30s
+        interval: 1m
+        retries: 3
+
+  ogree-bff:
+    build:
+      context: ${CORE_DIR}
+      dockerfile: ${BFF_BUILD_DIR}/Dockerfile
+    image: ogree-bff:${IMAGE_TAG}
+    restart: on-failure:10
+    container_name: ${COMPOSE_PROJECT_NAME}_bff
+    environment:
+      - ENV=production
+      - BFF_PORT=${BFF_PORT}
+      - BFF_SECRET=${AUTH_SECRET}
+    volumes:
+      - ./bff_api_list.json:/bin/api.json
+    depends_on:
+      - ogree_api
+      - arango_api
+    ports:
+      - ${BFF_PORT}:${BFF_PORT}
+
 volumes:
-  db:
\ No newline at end of file
+  db:
+  arangodb-persist:
+    external: true
```

Comment:
Is it needed ? it forces us to do docker volume create manually, whereas it would be created automatically by default

Reply:
Not necessary, it's can be remove